deepchem/molnet/load_function/bbbc_datasets.py

```diff
@@ -146,3 +150,113 @@ def load_bbbc002(
     loader = _BBBC002Loader(featurizer, splitter, transformers, BBBC2_TASKS,
                             data_dir, save_dir, **kwargs)
     return loader.load_dataset('bbbc002', reload)
+
+
+class _BBBC004_Segmentation_Loader(_MolnetLoader):
+
+    def __init__(self, overlap_probability: float = 0.0, **kwargs):
+        overlap_dict = {0.0: "00", 0.15: "15", 0.3: "30", 0.45: "45", 0.6: "60"}
```

Comment:
How do i mention the data type for inputs from a given list? 
Currently, I just accept floats and reject everything outside the given options and mention the same in the docs. 
Please let me know so that I can make the appropriate changes. 
@rbharath @shreyasvinaya